api/src/infrastructure/kubernetes/payloads.rs

```diff
@@ -251,6 +248,26 @@ pub fn deployment_payload(
     .expect("Cannot convert value to apps/v1/Deployment")
 }
 
+/// Creates an annotation object so that the underlying pod will be deployed according to its
+/// deployment strategy.
+///
+/// For example, this [popular workaround](https://stackoverflow.com/a/55221174/5088458) will be
```

Comment:
The information here is very confusing as it only is part of RedeployAlways. I would comment where the workaroud is used, so in line 265 or at least make it clear here, that only that part is meant here. So I would say somethink like:
```
 A [popular workaround](https://stackoverflow.com/a/55221174/5088458)  is applied to ensure that for DeploymentStrategy::RedeployAlways k8s will always update the deployment.
```